runtime/src/lib.rs

```diff
@@ -399,6 +400,71 @@ impl pallet_scheduler::Config for Runtime {
     type Preimages = Preimage;
 }
 
+parameter_types! {
+    pub const TreasuryPalletId: PalletId = PalletId(*b"zk/trsry");
+    pub const ProposalBond: Permill = Permill::from_percent(5);
+    pub const ProposalBondMinimum: Balance = 2000 * CENTS;
+    pub const ProposalBondMaximum: Balance = THOUSANDS;
+    pub const SpendPeriod: BlockNumber = 6 * DAYS;
+    pub const Burn: Permill = Permill::from_percent(1);
+    pub const PayoutSpendPeriod: BlockNumber = 30 * DAYS;
+    pub const MaxApprovals: u32 = 100;
+    pub ZKVerifyTreasuryAccount: AccountId = TreasuryPalletId::get().into_account_truncating();
+}
+
+impl pallet_treasury::Config for Runtime {
+    type PalletId = TreasuryPalletId;
+    type Currency = Balances;
+    type ApproveOrigin = EitherOfDiverse<EnsureRoot<AccountId>, Treasurer>;
+    type RejectOrigin = EitherOfDiverse<EnsureRoot<AccountId>, Treasurer>;
+    type RuntimeEvent = RuntimeEvent;
+    type OnSlash = Treasury;
+    type ProposalBond = ProposalBond;
+    type ProposalBondMinimum = ProposalBondMinimum;
+    type ProposalBondMaximum = ProposalBondMaximum;
+    type SpendPeriod = SpendPeriod;
+    type Burn = Burn;
+    type BurnDestination = ();
+    type MaxApprovals = MaxApprovals;
+    type WeightInfo = weights::pallet_treasury::ZKVWeight<Runtime>;
+    type SpendFunds = Bounties;
+    type SpendOrigin = TreasurySpender;
+    type AssetKind = ();
+    type Beneficiary = AccountId;
+    type BeneficiaryLookup = IdentityLookup<Self::Beneficiary>;
+    type Paymaster = PayFromAccount<Balances, ZKVerifyTreasuryAccount>;
+    type BalanceConverter = UnityAssetBalanceConversion;
+    type PayoutPeriod = PayoutSpendPeriod;
+    #[cfg(feature = "runtime-benchmarks")]
+    type BenchmarkHelper = ();
+}
+
+parameter_types! {
+    pub const BountyDepositBase: Balance = ACME;
+    pub const BountyDepositPayoutDelay: BlockNumber = 8 * DAYS;
+    pub const BountyUpdatePeriod: BlockNumber = 90 * DAYS;
+    pub const MaximumReasonLength: u32 = 16384;
+    pub const CuratorDepositMultiplier: Permill = Permill::from_percent(50);
+    pub const CuratorDepositMin: Balance = 10 * ACME;
+    pub const CuratorDepositMax: Balance = 200 * ACME;
+    pub const BountyValueMinimum: Balance = 10 * ACME;
+    pub DataDepositPerByte: Balance = deposit(0, 1);
+}
+impl pallet_bounties::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type BountyDepositBase = BountyDepositBase;
+    type BountyDepositPayoutDelay = BountyDepositPayoutDelay;
+    type BountyUpdatePeriod = BountyUpdatePeriod;
+    type CuratorDepositMultiplier = CuratorDepositMultiplier;
+    type CuratorDepositMin = CuratorDepositMin;
+    type CuratorDepositMax = CuratorDepositMax;
+    type BountyValueMinimum = BountyValueMinimum;
+    type ChildBountyManager = ();
```

Comment:
I'm afraid we might need also the ChildBounties pallet. Bounties are usually used to access relevant portion of Treasury funds and allocate them to some activities to be specified later via child bounties. Bounties are an easier way to access Treasury Funds when compared to OpenGov referenda. More info [here](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury#bounties).

Reply:
added